src/main/cpp/src/utils/vid_mapper.cc

```diff
@@ -416,18 +418,25 @@ const {
   types.push_back(std::type_index(typeid(int64_t)));
 
   std::vector<int> compression;
```

Comment:
Call this field compression_types instead of compression too to be consistent with compression_levels